### PR TITLE
started readme, still need a screenshot and link to deployed application

### PR DESCRIPTION
**Pull request by jpboo888**
started readme, just need a screenshot and link to deployed application

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+- - -
+© 2022 Trilogy Education Services, LLC, a 2U, Inc. brand. Confidential and Proprietary. All Rights Reserved.
```
Comment (hellovolcano) [inline README.md:46]: This (and the preceding license) should probably be removed, right?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Materialize framework
+
+## Screenshot & Link
+
+![screenshot](./assets/images/portfolio%20screen%20shot.png)
+https://jpboo888.github.io/portfolio1/
+
+## Contributing
+
+Made with ❤️ by Juan Pablo Maldonado, Valerie Gleason, & Mogwai Turner
```
Comment (hellovolcano) [inline README.md:21]: Maybe we could link to each of our github profiles or deployed portfolios here? Mine is https://hellovolcano.github.io/portfolio

**Review by hellovolcano — COMMENTED** [review]
I added some feedback/suggestions, but I don't  feel like I need to review it again if you make the fixes.

**Review by MogwaiTurner — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (MogwaiTurner) [reply README.md:21]: Mine still needs work, but this is my portfolio so far. https://mogwaiturner.github.io/Portfolio/

**Review by hellovolcano — APPROVED** [review]

**Pull request merged**